flutter/lib/src/navigation/sentry_navigator_observer.dart

```diff
@@ -44,7 +44,7 @@ class SentryNavigatorObserver extends RouteObserver<PageRoute<dynamic>> {
   final Hub hub;
 
   @override
-  void didPush(Route<dynamic> route, Route<dynamic> previousRoute) {
+  void didPush(Route<dynamic> route, Route<dynamic>? previousRoute) {
```

Comment:
The first route arguments in `didPush`, `didReplace` and `didPop` seem inconsistent. Are you sure it's right?

Reply:
Yes. I look at parent classes:

![image](https://user-images.githubusercontent.com/5236035/110188690-94e19b80-7e57-11eb-90eb-eb34019ac2b7.png)
![image](https://user-images.githubusercontent.com/5236035/110188700-9b701300-7e57-11eb-981c-09d17969d99b.png)